common.go

```diff
@@ -28,11 +29,19 @@ func getCopyOptions(store storage.Store, reportWriter io.Writer, sourceSystemCon
 	if destinationSystemContext != nil {
 		*destinationCtx = *destinationSystemContext
 	}
+	if removeSignatures {
+		logrus.Debugf("removing signatures while copying image")
```

Comment:
Is this a placeholder?

Reply:
It was, before I started writing tests.  I'll drop them.